watertap/unit_models/MD/membrane_distillation_1D.py

```diff
@@ -0,0 +1,548 @@
+
+        .. csv-table::
+            :header: "Configuration Options", "Description"
+
+            "``FrictionFactor.flat_sheet``", "Friction factor correlation for flat-sheet membrane modules"
+            "``FrictionFactor.spiral_wound``", "Friction factor correlation for spiral-wound membranes"
+        """,
+        ),
+    )
+
+    _CONFIG_Template.declare(
+        "property_package_vapor",
+        ConfigValue(
+            default=useDefault,
+            domain=is_physical_parameter_block,
+            description="Property package to use for control volume",
+            doc="""Property parameter object used to define property calculations,
+        **default** - useDefault.
+        **Valid values:** {
+        **useDefault** - use default package from parent model or flowsheet,
+        **PhysicalParameterObject** - a PhysicalParameterBlock object.}""",
+        ),
+    )
+    _CONFIG_Template.declare(
+        "property_package_args_vapor",
+        ConfigBlock(
+            implicit=True,
+            description="Arguments to use for constructing property packages",
+            doc="""A ConfigBlock with arguments to be passed to a property block(s)
+        and used when constructing these,
+        **default** - None.
+        **Valid values:** {
+        see property package for documentation.}""",
+        ),
+    )
+
+    CONFIG.declare("hot_ch", _CONFIG_Template(doc="hot channel config arguments"))
+    CONFIG.declare("cold_ch", _CONFIG_Template(doc="cold channel config arguments"))
+
+    # Common config args for both sides
+
+    CONFIG.declare(
+        "area_definition",
+        ConfigValue(
+            default=DistributedVars.uniform,
+            domain=In(DistributedVars),
+            description="Argument for defining form of area variable",
+            doc="""Argument defining whether area variable should be spatially
+        variant or not. **default** - DistributedVars.uniform.
+        **Valid values:** {
+        DistributedVars.uniform - area does not vary across spatial domain,
+        DistributedVars.variant - area can vary over the domain and is indexed
+        by time and space.}""",
+        ),
+    )
+    CONFIG.declare(
+        "transformation_method",
+        ConfigValue(
+            default=useDefault,
+            description="Discretization method to use for DAE transformation",
+            doc="""Discretization method to use for DAE transformation. See
+        Pyomo documentation for supported transformations.""",
+        ),
+    )
+    CONFIG.declare(
+        "transformation_scheme",
+        ConfigValue(
+            default=useDefault,
+            description="Discretization scheme to use for DAE transformation",
+            doc="""Discretization scheme to use when transformating domain. See
+        Pyomo documentation for supported schemes.""",
+        ),
+    )
+    CONFIG.declare(
+        "finite_elements",
+        ConfigValue(
+            default=20,
```

Comment:
Any reason for bumping this up to 20?

Reply:
Based on previous MD model experiences, changes in results are negligible beyond this point.

Reply:
That makes sense--although I still question whether the default needs to be that value as opposed to a lower value that would solve quicker. I would say if the time to solve isn't too lengthy (for exploratory purposes) at 20 elements, then this is fine. I could see the case where we might want to reduce this value for some tests that will constantly rerun with every PR. 

Whatever the case may be, we can save this for a future PR if we think it would be best to make any adjustments (either in tests or directly to default).